Tests/NetworkServiceTests/HTTPTaskTests.swift

```diff
@@ -0,0 +1,127 @@
+import NetworkService
+import XCTest
+
+final class HTTPTaskTests: XCTestCase {
+    func testHTTPTaskCasesPresentInAPI() {
+        _ = HTTPTask.request
+        _ = HTTPTask.requestWithURLParameters(urlParameters: nil)
+        _ = HTTPTask.requestWithBodyParameters(bodyParameters: nil)
+    }
+
+    func testCreatesHTTPTaskWithEmptyURLParameters() {
+        // Given
+        let emptyUrlParameters = URLParameters()
+
+        // When
+        let task = HTTPTask.requestWithURLParameters(urlParameters: emptyUrlParameters)
+
+        // Then
+        XCTAssertNotNil(task)
+    }
+
+    func testCreatesHTTPTaskWithNilAsURLParameters() {
+        // Given
+        let nilUrlParameters: URLParameters? = nil
+
+        // When
+        let task = HTTPTask.requestWithURLParameters(urlParameters: nilUrlParameters)
```

Comment:
Do we really need to create an object with a nil value?
```suggestion
        // Given,  When
        let task = HTTPTask.requestWithURLParameters(urlParameters: nil)
```

Reply:
Fixed